<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeDialogBase</class>
 <widget class="QDialog" name="RangeDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>149</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>149</horstretch>
    <verstretch>115</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>10</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>149</width>
    <height>123</height>
   </size>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>Messbereiche</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="RangeNComboBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="RangeEVTComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="RangeXTextLabel">
     <property name="text">
      <string>&amp;X:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>RangeXComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="RangeXComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="RangeEVTTextLabel">
     <property name="text">
      <string>&amp;EVT:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>RangeEVTComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="RangeNTextLabel">
     <property name="text">
      <string>&amp;N:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>RangeNComboBox</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <includes>
  <include location="local">wmglobal.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>RangeNComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>RangeDialogBase</receiver>
   <slot>SetNRangeSlot(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RangeXComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>RangeDialogBase</receiver>
   <slot>SetXRangeSlot(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RangeEVTComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>RangeDialogBase</receiver>
   <slot>SetEVTRangeSlot(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
